### PR TITLE
Replace try/catch with annotations in tests

### PR DESCRIPTION
**Pull request by shadowhand**
Annotations are easier to configure and result in less overall code.

Refs #2

### CODE REVIEW

**Comment by elazar** [comment]
Maybe we should instead switch to [this practice](https://thephp.cc/news/2016/02/questioning-phpunit-best-practices)?

**Comment by shadowhand** [comment]
We should indeed. @elazar

**Comment by elazar** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/equip/auth/pull-request/3/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=equip/auth)

**Comment by elazar** [comment]
@shadowhand Merge conflicts.

File: tests/Jwt/FirebaseParserTest.php [hunk 1]
```diff
@@ -51,32 +50,28 @@ protected function setUp()
         ];
     }
 
+    /**
+     * @expectedException \Equip\Auth\Exception\InvalidException
+     * @expectedExceptionCode \Equip\Auth\Exception\InvalidException::CODE_TOKEN_EXPIRED
```
Comment (shadowhand) [inline tests/Jwt/FirebaseParserTest.php:55]: We should use `setExpectedException` (or similar) instead of using annotations.

Comment (elazar) [inline tests/Jwt/FirebaseParserTest.php:55]: Anything stopping us from upgrading to PHPUnit >= 5.2 and transitioning to [this practice](https://thephp.cc/news/2016/02/questioning-phpunit-best-practices) as I [suggested earlier](https://github.com/equip/auth/pull/3#issuecomment-181497866)?

Comment (AlexMasterov) [inline tests/Jwt/FirebaseParserTest.php:55]: @elazar This version of PHPUnit requires PHP 5.6. The Equip still supports PHP 5.5 (Travis CI).

[Supported Versions of PHP](https://secure.php.net/supported-versions.php)

Comment (elazar) [inline tests/Jwt/FirebaseParserTest.php:55]: @AlexMasterov Fair enough, though dropping 5.5 support may be something we want to consider doing given that it's going EOL next month.

**Comment by shadowhand** [comment]
This has conflicts and cannot be merged.

**Pull request closed without merging**